aquacrop/initialize/read_model_parameters.py

```diff
@@ -130,6 +131,16 @@ def read_model_parameters(
     single_year = pd.to_datetime("1990/" + crop.planting_date) < pd.to_datetime(
         "1990/" + crop.harvest_date
     )
+
+    # Check if the simulation in the following year does not reach the planting date.
+    mock_simulation_end_date = pd.to_datetime("1990/" + f'{sim_end_date.month}' + "/" + f'{sim_end_date.day}')
+    mock_simulation_start_date = pd.to_datetime("1990/" + crop.planting_date)
+    last_simulation_year_does_not_start = mock_simulation_end_date < mock_simulation_start_date
+    if single_year and last_simulation_year_does_not_start:
+        start_end_years = pd.DatetimeIndex([sim_start_date, sim_end_date - relativedelta(years=1)]).year
```

Comment:
I would change the variable start_end_years. It can be a normal array like:
```python
if single_year and last_simulation_year_does_not_start:
        start_end_years = [sim_start_date.year, sim_end_date.year - 1]
else:
        start_end_years =[sim_start_date.year, sim_end_date.year]

```

Reply:
Or even a better solution would be to put it like this
```python
start_end_years = [sim_start_date.year, sim_end_date.year]

if single_year:
  # if normal year
  # Check if the simulation in the following year does not reach the planting date.
  mock_simulation_end_date = pd.to_datetime("1990/" + f'{sim_end_date.month}' + "/" + f'{sim_end_date.day}')
  mock_simulation_start_date = pd.to_datetime("1990/" + crop.planting_date)
  last_simulation_year_does_not_start = mock_simulation_end_date < mock_simulation_start_date
  
  if last_simulation_year_does_not_start:
      start_end_years[1] = start_end_years[1] -1
```